_includes/layout.njk

```diff
@@ -1,6 +1,7 @@
 ---
-showregisterbutton: true
+showregisterbutton: false
```

Comment:
I set showregisterbutton to false.  This hides it on the bottom of every page (from layout)